### DISCUSSION TITLE
[ 灌水 ] 可恶的远景实业！！

### DISCUSSION THREAD

**Discussion by share121** [share121/inter-knot#17] [其他]
最近几个月的生意太不顺了！
前段时间我出委托时骨折了，好不容易养好病，之前熟悉的老客户也都有了新的绳匠搭档。我费劲千辛万苦才从绳网中介那接到一个勘探的活，结果到了今天事情做完该结款了，我才知道委托是远景实业发的！
现在远景实业的项目停了，绳网上所有他们找人发布的委托都撤销了！我只能象征性地拿到20%的违约金！鬼知道这该死的委托到底有多麻烦！我这一个月的时间都搭在上面了！

**Comment by share121** [comment]
别看远景实业财大气粗，可是出了名的抠门企业，我之前也被他们坑过。中介骗我说委托是一个高校研究者发起的，等做完拿到钱才知道是远景。
拜托，绳网有绳网的规矩，是企业的委托就得给企业的价，冒充私人压价是什么意思？

**Comment by share121** [comment]
楼上，你好歹还顺利拿到了钱。
最近我刚为了接远景的活儿推掉了两个别的委托现在那两个委托…已经有别的绳匠接了，我这个月要颗粒无收了。
本来我这种菜鸟绳匠，想找到活就很艰难了，唉。

**Comment by share121** [comment]
像三门、圣霖、朽峰这些企业，好列还是会按绳网的规矩找人办事，至于远景嘛…不过之后大家也再也不会被远景骗了，这公司就算不倒闭，没个十年八年也别想东山再起了。

**Comment by share121** [comment]
之前好像有过远景外包的工程队在空洞里出了问题，远景不肯找调查协会救援，而是在绳网找人去救的事情吧？

**Comment by share121** [comment]
我觉得在绳网求助也没什么大问题，毕竟调查协会的效率也不一定高，绳网上高手大有人在。

**Comment by ssdxs15** [comment]
我和我自己聊天

**Comment by Zzz-walnut** [comment]
与其在远景继续做牛做马，不如来白祇重工，业界新星福利好待遇高，听说女神格莉丝也在
